README.md

```diff
@@ -158,6 +158,7 @@ All configuration options are controlled through environment variables.
 * `DEYE_LOGGER_IP_ADDRESS` - inverter data logger IP address
 * `DEYE_LOGGER_PORT` - inverter data logger communication port, typically 8899
 * `DEYE_FEATURE_MQTT_PUBLISHER` - controls, if the service will publish metrics over mqtt, defaults to `true`
+* `DEYE_FEATURE_STDOUT_PUBLISHER` - controls, if the service will publish metrics on stdout, defaults to `false`
```

Comment:
Please add this env var in `config.env.example` too

Reply:
Done.
Took the liberty to also include the other feature gates.